api/src/opentrons/commands/module_commands.py

```diff
@@ -150,3 +152,64 @@ def thermocycler_wait_for_lid_temp() -> command_types.ThermocyclerWaitForLidTemp
 def thermocycler_close() -> command_types.ThermocyclerCloseCommand:
     text = "Closing Thermocycler lid"
     return {"name": command_types.THERMOCYCLER_CLOSE, "payload": {"text": text}}
+
+
+def heater_shaker_set_target_temperature(
+    celsius: float,
+) -> command_types.HeaterShakerSetTargetTemperatureCommand:
+    formatted_temp = trunc(celsius)
+    text = f"Setting Target Temperature of Heater-Shaker to {formatted_temp} °C"
+    return {
+        "name": command_types.HEATER_SHAKER_SET_TARGET_TEMPERATURE,
+        "payload": {"text": text},
+    }
+
+
+def heater_shaker_wait_for_temperature() -> command_types.HeaterShakerWaitForTemperatureCommand:
+    text = "Waiting for Heater-Shaker to reach target temperature"
+    return {
+        "name": command_types.HEATER_SHAKER_WAIT_FOR_TEMPERATURE,
+        "payload": {"text": text},
+    }
+
+
+def heater_shaker_set_and_wait_for_shake_speed(
+    rpm: int,
+) -> command_types.HeaterShakerSetAndWaitForShakeSpeedCommand:
+    text = f"Setting Heater-Shaker to Shake at {rpm} RPM and waiting until reached"
```

Comment:
IMO we can shorten this to:

```suggestion
    text = f"Setting Heater-Shaker to shake at {rpm} RPM"
```

The fact that it waits until the RPM is reached is meaningful for engineers and protocol authors, but if you're just operating the OT-2 watching the run log, it might be a detail that's more confusing than helpful.

Reply:
In hardware documentation I've used lowercase `rpm`. Most dictionaries I've checked have the headword lowercase and [Google ngrams data](https://books.google.com/ngrams/graph?content=rpm&year_start=1900&year_end=2019&corpus=26&smoothing=3&case_insensitive=true) bears it out.

Reply:
So I like the explicit mention of 'waiting' since it distinguishes the command from the ones that only set the target. BUT, we don't mention it for other modules' blocking commands so I'm conflicted on this.